client/dom/GeneSetEdit/GeneSetEditUI.ts

```diff
@@ -188,8 +199,73 @@ export class GeneSetEditUI {
 		}
 		if (this.mode == TermTypes.GENE_EXPRESSION && this.vocabApi.termdbConfig?.queries?.topVariablyExpressedGenes) {
 			if (this.vocabApi.termdbConfig.queries.topVariablyExpressedGenes.arguments) {
-				for (const param of this.vocabApi.termdbConfig.queries.topVariablyExpressedGenes.arguments)
+				for (const param of this.vocabApi.termdbConfig.queries.topVariablyExpressedGenes.arguments) {
+					if (param.radiobuttons) {
+						//set a default value
+						if (!param.value)
+							param.value = {
+								type: param.radiobuttons[0].value,
+								geneList: null
+							}
+						for (const opt of param.radiobuttons) {
+							if (opt.type == 'tree') {
+								opt.callback = async (holder: Elem) => {
+									const termdb = await import('../../termdb/app.js')
+									termdb.appInit({
+										holder,
+										state: {
+											dslabel: opt.value,
+											genome: this.genome.name,
+											nav: {
+												header_mode: 'search_only'
+											}
+										},
+										tree: {
+											click_term: (term: any) => {
+												param.value = {
+													type: opt.value,
+													geneList: term._geneset.map((t: any) => t.symbol)
+												}
+											}
+										}
+									})
+								}
+							}
+							if (opt.type == 'text') {
+								opt.callback = async (holder: Elem) => {
+									holder
+										.append('span')
+										.style('display', 'block')
+										.style('font-size', '0.8em')
+										.style('opacity', 0.75)
+										.text('Enter genes separated by comma')
```

Comment:
please change to separated by space or comma

Reply:
Fixed in https://github.com/stjude/proteinpaint/pull/1883/commits/482fc1e8f0fe1e0bc05bc11e436e88ae3670473b